src/Renderer/Shader/PointsFS.glsl

```diff
@@ -1,17 +1,37 @@
-precision highp float;
-precision highp int;
-
+#include <itowns/precision_qualifier>
 #include <logdepthbuf_pars_fragment>
+#if defined(USE_TEXTURES_PROJECTIVE)
+#include <itowns/projective_texturing_pars_fragment>
+#endif
 
 varying vec4 vColor;
-
+uniform bool pickingMode;
 void main() {
+    #include <logdepthbuf_fragment>
     // circular point rendering
     if(length(gl_PointCoord - 0.5) > 0.5){
         discard;
     }
 
-    gl_FragColor = vColor;
+#if defined(USE_TEXTURES_PROJECTIVE)
+    if (!pickingMode) {
+        vec4 color = vColor;
+        vec4 aColor;
+        float alpha = 0.0;
 
-    #include <logdepthbuf_fragment>
+        #pragma unroll_loop
+        for (int i = 0; i < ORIENTED_IMAGES_COUNT; i++) {
```

Comment:
Maybe factorize this part with `ProjectiveTextureFS.glsl` ? It is almost the same, beside the `#ifdef USE_BASE_MATERIAL`.

Reply:
Factorized